lints/mozilla/lint_mp_ecdsa_allowed_curve_hash_pair.go

```diff
@@ -0,0 +1,131 @@
+/*
+ * ZLint Copyright 2019 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+/********************************************************************
+Section 5.1 - Algorithms
+ECDSA keys using one of the following curve-hash pairs: P‐256 with SHA-256, P‐384 with SHA-384
+********************************************************************/
+
+package lints
+
+import (
+	"encoding/asn1"
+	"errors"
+	"fmt"
+	"math/big"
+
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zlint/lint"
+	"github.com/zmap/zlint/util"
+)
+
+type ecdsaSignature struct {
+	R, S *big.Int
+}
+
+func getSigningKeySize(cert *x509.Certificate) (int, error) {
+	sig := new(ecdsaSignature)
+	if _, err := asn1.Unmarshal(cert.Signature, sig); err != nil {
+		return -1, err
```

Comment:
Stylistic nitpick, 0 feels a bit more Go-y than -1 here. Typically I've seen the equivalent of nil, error.

Reply:
Agreed, addressed in 0828819